src/mono/wasm/runtime/loader/libraryInitializers.ts

```diff
@@ -30,6 +18,7 @@ async function fetchLibraryInitializers(config: MonoConfig, type: LibraryInitial
     async function importInitializer(path: string): Promise<void> {
         try {
             const adjustedPath = appendUniqueQuery(loaderHelpers.locateFile(path), "js-module-library-initializer");
+            mono_log_debug(`Attempting to import '${adjustedPath}' for ${path}`);
```

Comment:
Intended to keep this log message?

Reply:
yes, this is debug level and it will write only when `diagnosticTracing` is set.